errors.go

```diff
@@ -159,19 +159,21 @@ type withStack struct {
 
 func (w *withStack) Cause() error { return w.error }
 
+func (w *withStack) Unwrap() error { return w.error }
```

Comment:
The Unwrap method is already added on the PR #206 furthermore this PR will be merge soon so actually you wouldn't have to add it

Reply:
If I delete this, the unit test will be failed, however after merge PR #206 it will be success, so I keep the unit test and wait for PR #206